workload/metrics_full.yaml

```diff
@@ -1,6 +1,8 @@
 # API server
 
-# metrics:    # Not needed since kube-version >= 0.12
+{{if .KUBE_BURNER_RELEASE eq 0.11}}
```

Comment:
I am not sure if `.KUBE_BURNER_RELEASE` will be a Integer or a String. In the second scenario, just change for `eq "0.11"` will do the trick.